agent/dockerclient/dockerapi/errors.go

```diff
@@ -0,0 +1,248 @@
+// Copyright 2014-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package dockerapi
+
+import (
+	"time"
+
+	"github.com/aws/amazon-ecs-agent/agent/dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
+)
+
+const (
+	DockerTimeoutErrorName           = "DockerTimeoutError"
+	CannotInspectContainerErrorName  = "CannotInspectContainerError"
+	CannotDescribeContainerErrorName = "CannotDescribeContainerError"
+)
+
+// DockerTimeoutError is an error type for describing timeouts
+type DockerTimeoutError struct {
+	Duration   time.Duration
```

Comment:
I know you're just moving these between files. But, this is as good a time as any to add comments for these fields.